### PR TITLE
Feat: Add randomize option

### PR DESCRIPTION
**Pull request by changchanghwang**
### Development description

- Add randomize option type 
- https://github.com/tim-kos/node-retry/blob/11efd6e4e896e06b7873df4f6e187c1e6dd2cf1b/lib/retry.js#L22

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -85,6 +85,24 @@ export type Options = {
...
+	 import pRetry from 'p-retry';
+	 *
+	 const run = async () => { ... };
+	 *
+	 const result = await pRetry(run, {
+	   randomize: true
+	 });
+	 ```
+	 */
+	 readonly randomize?: boolean;
```
Comment (sindresorhus) [inline index.d.ts:105]: The option is coming from the `retry` package, so this needs to be added to `@types/retry`.

File: index.d.ts [hunk 2]
```diff
@@ -85,6 +85,24 @@ export type Options = {
 	```
 	*/
 	readonly signal?: AbortSignal;
+
+	/**
+	 Decide if a random delay should be introduced between retries.
+	 Enabling this option adds variability to the retry strategy, helping to distribute load peaks and reduce contention.
+	 If set to `true`, a random delay will be introduced between each retry attempt.
```
Comment (sindresorhus) [inline index.d.ts:92]: I needs to say what the random delay range could be.

File: readme.md [hunk 3]
```diff
@@ -143,6 +143,23 @@ try {
...
+
+```js
+import pRetry from 'p-retry';
+
+const run = async () => { … };
+
+const result = await pRetry(run, {
+	randomize: true
+});
+```
```
Comment (sindresorhus) [inline readme.md:160]: This is already covered by:

> Options are passed to the [`retry`](https://github.com/tim-kos/node-retry#retryoperationoptions) module.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by changchanghwang** [comment]
@sindresorhus 
sorry for wasting your time :( 
I didn't see `& OperationOptions`.. this pull request isn't necessary anymore.

**Pull request closed without merging**